src/smp/lock.c

```diff
@@ -4,15 +4,19 @@
  * SPDX-License-Identifier: GPL-2.0-only
  */
 
+#include <assert.h>
```

Comment:
please leave `config.h` has the first include and include `assert.h` afterwards

Reply:
Done.